docs/docs/en/architecture/configuration.md

```diff
@@ -187,6 +187,9 @@ The default configuration is as follows:
 
 Note that DolphinScheduler also supports zookeeper related configuration through `bin/env/dolphinscheduler_env.sh`.
 
+For ETCD Registry, please see more details on [link](https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-etcd/README.md).
```

Comment:
``` markdown
## Task Parameters

[//]: # (TODO: use the commented anchor below once our website template supports this syntax)
[//]: # (- Please refer to [DolphinScheduler Task Parameters Appendix]&#40;appendix.md#default-task-parameters&#41; `Default Task Parameters` section for default parameters.)

```

This reminds me of an old issue. As the comment shows above, I'm wondering whether our website has supported markdown anchor? @zhongjiajie